.changeset/red-ants-lick.md

```diff
@@ -0,0 +1,18 @@
+---
+'xstate': minor
+---
+
+author: @Andarist
+author: @mattpocock
+
+Added the ability to tighten TS declarations of machine with generated metadata. This opens several exciting doors to being able to use typegen seamlessly with XState to provide an amazing typing experience.
+
+With the [VSCode extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode), you can specify a new attribute called `tsTypes: true` in your machine definition:
```

Comment:
the URL here will require to be updated, right? I've seen that you were setting up Stately publisher with David

Reply:
Yep, we can adjust later

Reply:
Sure, just remember about updating that before releasing the new version of XState - people subscribe to our releases and it would be preferable if the final URL would end up in the emails sent out to them.